### DISCUSSION TITLE
Do not render `inspect_args` when `env` is set to `production`

### DISCUSSION THREAD

**Discussion by auryark** [bashly-framework/bashly#479] [Ideas]
I believe that the existence and use of this function is more suitable for those who are **developing** a project using Bahsly, but I don't see a case in which it would be useful for anyone who is actually **using** the generated script.

**Comment by DannyBen** [comment]
Interesting point.
Any other functions to remove in your opinion?

**Reply by auryark** [reply]
Based on my current knowledge, I don't see another one being removed, but that could be open to other opinions.

**Reply by DannyBen** [reply]
I will take a deeper look and implement at least the removal of `inspect_args`. Good catch.

*Converting this to issue*
